Lesson2/src/main/java/pl/solvd/university/administration/Accounting.java

```diff
@@ -0,0 +1,88 @@
+package pl.solvd.university.administration;
+
+import pl.solvd.university.state.Payment;
+import pl.solvd.university.state.ReturnDocuments;
+
+import java.io.IOException;
+import java.util.Locale;
+import java.util.Scanner;
+
+import static pl.solvd.university.Main.*;
+import static pl.solvd.university.documents.Pass.show;
+
+public class Accounting {
+    public enum Bill {
+        EDUCATION(2000);
+        private final int price;
+
+        Bill(int price) {
+            this.price = price;
+        }
+
+        public int getPrice() {
+            return price;
+        }
+    }
+
+    static void exceptPayment(Bill category) throws Exception {
+        System.out.printf("Accounting:\nNEW BILL:\n%s payment is %d dollars.\n", category.name(), category.getPrice());
+        System.out.println("Ready to make a payment?");
+        Scanner answer = new Scanner(System.in);
+        String chooseDone = answer.next();
+        if ((!(chooseDone.toUpperCase(Locale.ROOT).equals("YES")) && (!(chooseDone.toUpperCase(Locale.ROOT).equals("NO"))))) {
+            try {
+                throw new IOException("Exception: Something went wrong. Please, try to pay again");
+            } catch (IOException e) {
+                LOG.error("Exception: Something went wrong. Invalid input. Try to pay again");
+                System.out.println(e.getMessage());
+                exceptPayment(Bill.EDUCATION);
+                applicant.save();
+            }
+        }
+            else{
+                switch (chooseDone.toUpperCase(Locale.ROOT)) {
+                    case "NO" -> {
+                        LOG.info("Applicant refused to pay now");
+                        System.out.println("Payment declined. Please try to pay later");
+                    }
+                    case "YES" -> {
+                        LOG.info("Payment was successful");
+                        System.out.println("Payment accepted");
+                    }
+                }
+            }
+        }
+
+        static void agreement (Bill category) throws Exception {
```

Comment:
```suggestion
        static void agreement(Bill category) throws Exception {
```
How does `category` can be a Bill?

Reply:
method provides for the acceptance and payment of payments in different categories: one of them is Education. Maybe for a hostel, for a lost book in the library, a scholarship payment. However, yes, at the moment there is only one payment option - for education

Reply:
It is just about naming. paymentCategory / billCategory / paymentDestination or smth like that.

Reply:
Done